scripts/run.sh

```
@@ -0,0 +1,224 @@
+    done
+    wait 2>/dev/null
+    exit 0
+}
+trap cleanup INT TERM
+
+start_backend() {
+    (
+        echo "[BACKEND] Starting... ${BACKEND_DIR}"
+        cd "$BACKEND_DIR"
+        bin_dir=$(resolve_venv "BACKEND" "$BACKEND_DIR" ".env" "venv") || exit 1
+        export PATH="$bin_dir:$PATH"
```

Shellcheck (0.11.0)

[info] Modification of PATH is local (to subshell caused by (..) group).

(SC2030)